pkg/util/winutil/users_test.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018-present Datadog, Inc.
+
+// +build windows
+
+package winutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetSidFromUser(t *testing.T) {
+	sid, err := GetSidFromUser()
+	t.Logf("The SID found was: %v", sid)
+	assert.Nil(t, err)
+}
+
+func TestGetUserFromSid(t *testing.T) {
+	sid, err := GetSidFromUser()
+	assert.Nil(t, err)
+
+	username, domain, err := GetUserFromSid(sid)
```

Comment:
Some assertions around the value/type of `username/domain` would be good otherwise we're just testing the `err` return value.